src/applications/vaos/appointment-list/redux/reducer.js

```diff
@@ -324,31 +324,25 @@ export default function appointmentsReducer(state = initialState, action) {
           return appt;
         }
 
-        const newAppt = {
-          ...appt,
-          vaos: {
-            ...appt.vaos,
-            apiData: action.apiData,
-          },
-        };
-
-        return { ...newAppt, status: APPOINTMENT_STATUS.cancelled };
+        return action.updatedAppointment;
```

Comment:
For requests, we have the new, transformed request passed back in the action now, so use that instead of making updates to existing data.